README.md

```diff
@@ -20,14 +20,16 @@ The Docker compose file at `test/docker-compose.yml` can be used to run the requ
 
 | name | part of | description | required | default value |
 |------|---------|-------------|----------|---------------|
-|`client_id`|source|The Client ID for Marketo Instance|true| |
-|`client_secret`|source|The Client Secret for Marketo Instance|true| |
-|`endpoint`|source|The Endpoint for Marketo Instance|true||
+|`ClientID`|source|The Client ID for Marketo Instance|true| |
+|`ClientSecret`|source|The Client Secret for Marketo Instance|true| |
+|`ClientEndpoint`|source|The Endpoint for Marketo Instance|true||
 |`polling period`|source|Polling time for CDC mode|false|`1m`|
+|`fields`|source|Fields to fetch from Leads database|false|`id, createdAt, updatedAt, firstName, lastName, email`|
 
 ### Known Issues & Limitations
-* Bulk API has some Limitations.
+#### Snapshot mode:
+* In snapshot mode, the total amount of data that you can export from Marketo is limited to 500MB per day unless you have purchased a higher data limit. This 500MB limit resets daily at 12:00AM CST.
```

Comment:
So what happens in that case? Do we continue on the next day?

Reply:
Can you please elaborate that "no actions are taken"? Does it mean the pipeline stops with an error, or does it mean that the next day we try again (but also fail again)?

Reply:
when 500mb limit is hit, pipeline stops with error, if pipeline is restarted again it will stop with same error. But if we try next day it will start pulling data from where it is left

Reply:
That's some important info, it would be good to have it here (in the ReadMe).

Reply:
I see that this convo has been resolved, but see no changes in the ReadMe?

Reply:
updated.